msteams-platform/tabs/what-are-tabs.md

```diff
@@ -14,59 +14,45 @@ Tabs are client-aware webpages embedded in Microsoft Teams, Outlook, and Microso
 > [!IMPORTANT]
 > Custom tabs are available in Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD).
 
-The following image shows static tabs:
+There are two types of tabs, static and configurable tabs. These tabs can be utilized in three distinct scopes: `personal`, `groupchat`, and `teams`. [Personal (static) tabs](~/tabs/how-to/create-personal-tab.md), along with personal-scoped bots, are part of the personal apps that are scoped to a single user and pinned to the left navigation bar for easy access. Personal (static) tabs in personal scope continue to work the same way. However, you can now [extend static tabs](~/tabs/how-to/create-personal-tab.md#extend-static-tabs-to-group-chat-channels-and-meetings) to group chat or meetings with a [customizable experience.](~/tabs/how-to/create-personal-tab.md#customizing-your-static-tab-in-chats-or-meetings)
 
-# [Desktop](#tab/desktop)
+You can extend static tabs to different scopes. Tabs in chats, channels, or meetings behave more like apps, as you can pin only one tab per app. For example, you can only pin a single YouTube app tab in a meeting. Static tabs in meetings can also be prepinned in meetings by IT Admins.
```

Comment:
This line is repeating the same from line 17 above. Recommend removing the first sentence ("You can extend static tabs to different scopes.") since this repeated above...
> However, you can now extend static tabs to group chats, channels or meetings with a customizable experience.

Reply:
Updated